CounterStrike2GSI/Nodes/Provider.cs

```diff
@@ -27,12 +27,18 @@ public class Provider : Node
         /// </summary>
         public readonly string SteamID;
 
+        /// <summary>
+        ///  Timestamp of the GameState data.
+        /// </summary>
+        public readonly long Timestamp;
```

Comment:
Based on the other comment, this should be an `int`.

```suggestion
        public readonly int Timestamp;
```